website/static/js/wikiTreebeard.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+var Fangorn = require('js/fangorn');
+var $ = require('jquery');
+var bootbox = require('bootbox');
+var m = require('mithril');
+var Treebeard = require('treebeard');
+var $osf = require('js/osfHelpers');
+require('../css/fangorn.css');
+
+
+function resolveToggle(item) {
+    var toggleMinus = m('i.fa.fa-minus', ' '),
+        togglePlus = m('i.fa.fa-plus', ' ');
+
+    if (item.children.length > 0) {
+        if (item.open) {
+            return toggleMinus;
+        }
+        return togglePlus;
+    }
+    item.open = true;
+    return '';
+}
+
+function resolveIcon(item) {
+    if (item.children.length > 0) {
+        if (item.open) {
+            return m('i.fa.fa-folder-open', ' ');
+        }
+        return m('i.fa.fa-folder', ' ');
+    }
+}
+
+function expandOnLoad() {
+    var tb = this;  // jshint ignore: line
+    for (var i = 0; i < tb.treeData.children.length; i++) {
+        var parent = tb.treeData.children[i];
+        if (parent.data.title === 'Project Wiki Pages') {
+            tb.updateFolder(null, parent);
+        }
+    }
+}
+
+function WikiMenu(data) {
+
+    //  Treebeard version
+    var tbOptions = {
+        divID: 'grid',
+        filesData: data,
+        resolveToggle: resolveToggle,
+        paginate : false,       // Whether the applet starts with pagination or not.
+        paginateToggle : false, // Show the buttons that allow users to switch between scroll and paginate.
+        uploads : false,         // Turns dropzone on/off.
+        hideColumnTitles: true,
+        resolveIcon : resolveIcon,
+        columnTitles: function () {
+            return[{
+                title: 'Name',
+                width: '100%'
+            }]
+        },
+        onload: function(tree) {
+            var tb = this;
+            tb.wikiID = window.contextVars.wiki.wikiID;
+            tb.canEdit = window.contextVars.wiki.canEdit;
```

Comment:
This can potentially be confusing to have canEdit right under tb. Can you put all these into a wiki object? Like this:
```js
   tb. wiki = {
       wikiID : window.contextVars.wiki.wikiID,
       canEdit : window.contextVars.wiki.canEdit;
   }
```